virttest/qemu_devices/qcontainer.py

```diff
@@ -2108,6 +2110,12 @@ def define_hbas(qtype, atype, bus, unit, port, qbus, pci_bus, iothread,
             devices.append(protocol_node)
             devices.append(format_node)
             # Add filter node
+            if image_copy_on_read in ("yes", "on", "true"):
```

Comment:
```suggestion
            if image_copy_on_read in ("yes", "on", True):
```
According the docstring of this method:
```
Creates related devices by variables
:note: To skip the argument use None, to disable it use False
:note: Strictly bool options accept "yes", "on" and True ("no"...)
```

Reply:
Hi Yongxue, image_copy_on_read is a string instead of a bool option, so what we pass to the function should never be a bool value.

Reply:
You are right, I forgot this point.

Reply:
Hi, Aihua
Can you paste the result of the part of the command? And run several cases to check the regression.
Thanks.

Reply:
(1/1) Host_RHEL.m9.u0.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.blockdev_commit_cor.q35: PASS (45.22 s)
(1/1) Host_RHEL.m9.u0.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.blockdev_stream_cor.q35: PASS (80.90 s)
(1/1) Host_RHEL.m9.u0.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.blockdev_stream_cor_base.q35: PASS (41.65 s)
(1/1) Host_RHEL.m9.u0.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.blockdev_commit_throttle.q35: PASS (65.49 s)

Reply:
Usage:
 -blockdev node-name=file_image1,driver=file,auto-read-only=on,discard=unmap,aio=threads,filename=/home/kvm_autotest_root/images/test-virtio-scsi.qcow2,cache.direct=on,cache.no-flush=off \
    -blockdev node-name=qcow2_image1,driver=qcow2,read-only=off,cache.direct=on,cache.no-flush=off,file=file_image1 \
    -blockdev node-name=drive_image1,driver=copy-on-read,file=qcow2_image1 \